tests/test_utils.py

```diff
@@ -17,6 +17,21 @@
     _PyperplanHeuristicWrapper
 
 
+def test_aabb_volume():
+    """Tests for get_aabb_volume."""
+    lo = np.array([1.0, 1.5, -1.0])
+    hi = np.array([2.0, 2.5, 0.0])
+    assert utils.get_aabb_volume(lo, hi) == 1.0
```

Comment:
add tests for zero volume and negative volume (lo < high, should hit our assert above)